api-contracts/openapi/components/schemas/event.yaml

```diff
@@ -115,6 +115,10 @@ EventWorkflowRunSummary:
       type: integer
       format: int64
       description: The number of failed runs.
+    cancelled:
+      type: integer
+      format: int64
+      description: The number of cancelled runs.
```

Comment:
nit but something I'm noticing in the code as I'm working on it now - this will never be null, right? since we have the `COALESCE`, so I imagine that it makes sense to mark this as `required` so we don't need to handle the null case in the TS

Reply:
good call 👍